assets/css/custom.css

```diff
@@ -0,0 +1,30 @@
+.story-preview-title{
```

Comment:
We should have all these classes in main.css as well with default values based on an english website

Reply:
Can you please suggest some good default font for English?